clusters/build-clusters/01_cluster/_machine_config/m5d_machineconfig.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: m5d
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      disks:
+      - device: "/dev/nvme1n1"
+        wipeTable: true
+        partitions:
+        - label: containerraid1
+          number: 0
+          start: 0
+          size: 0
+      - device: "/dev/nvme2n1"
+        wipeTable: true
+        partitions:
+        - label: containerraid2
+          number: 0
+          start: 0
+          size: 0
+      raid:
+      - devices:
+        - "/dev/disk/by-partlabel/containerraid1"
+        - "/dev/disk/by-partlabel/containerraid2"
+        level: stripe
+        name: containerraid
+      filesystems:
+      - mount:
+          device: "/dev/md/containerraid"
+          format: xfs
+          label: containers
+    systemd:
+      units:
+      - name: var-lib-containers.mount
+        enable: true
+        contents: |-
+          [Mount]
+          What=/dev/md/containerraid
+          Where=/var/lib/containers
+          Type=xfs
+
+          [Install]
+          WantedBy=local-fs.target
+      - dropins:
```

Comment:
You can drop the rest of the stuff here and below, this was just helpful for me testing locally to autologin on the console.

Reply:
Done.